filament/src/PostProcessManager.cpp

```diff
@@ -723,41 +724,48 @@ FrameGraphId<FrameGraphTexture> PostProcessManager::screenSpaceAmbientOcclusion(
      * The pass below that does this is automatically culled if not needed, which is decided by
      * each backend.
      */
+    auto const& depthDesc = fg.getDescriptor(depth);
+    FrameGraphId<FrameGraphTexture> duplicateDepthOutput;
+    if (mEngine.getDriverApi().isDepthStencilBlitSupported(depthDesc.format)) {
```

Comment:
At this point it might be better to either:
- move the DuplicateDepthPass into PostProcessManager (so that we get a simple `if{}else{}` here), or
- have a single blitDepth() method in PostProcessManager that picks the right implementation

Reply:
moved the conditional to `blitDepth`.